utils/node_local_test.py

```diff
@@ -1778,9 +1787,10 @@ def test_dfuse_dio_off(self):
             self.fatal_errors = True
 
     @needs_dfuse
-    def test_daos_fs_tool(self):
+    def Xtest_daos_fs_tool(self):
```

Comment:
Why does this need to be disabled?

Reply:
It was showing the memory leak which meant that future tests weren't running.  I was trying to get more coverage to see if there are other items we should know about.

Reply:
Re-enabled, and I've change the logic to report but carry on running in the presence of valgrind issues from the daos command, so that a single test-run can report all issues.